runtime/moonbase/src/lib.rs

```diff
@@ -1501,6 +1505,9 @@ mod benches {
 		[pallet_precompile_benchmarks, PrecompileBenchmarks]
 		[pallet_parameters, Parameters]
 		[pallet_xcm_weight_trader, XcmWeightTrader]
+		// pallet_collective instances
+		[pallet_collective, TreasuryCouncilCollective]
```

Comment:
Would be better to also rename this one:
```suggestion
		[pallet_collective_treasury_council, TreasuryCouncilCollective]
```